src/components/toggle/toggle.html

```diff
@@ -0,0 +1,13 @@
+<label [attr.for]="inputId" class="md-toggle-label">
+  <input #input class="md-toggle-input"
+         [type]="type"
+         [id]="inputId"
+         [checked]="checked"
+         [disabled]="disabled"
+         [name]="name"
+         (change)="onClick()"/>
```

Comment:
Got it from radio, sorry about that! Should there be a space between `"` and `>` or does it not matter?